R/retry.R

```diff
@@ -16,6 +20,10 @@
 #'   \code{pause_cap} seconds.
 #' @param quiet If \code{FALSE}, will print a message displaying how long
 #'   until the next request.
+#' @param terminate_on Optional vector of numeric or integer HTTP status codes
+#'   that if found on the response will immediately stop the retries. If missing
```

Comment:
I think the default value of NULL is not clear on what status codes will be terminated on the default scenario. I thought about setting the default value of `terminate_on` to `400:599`, but the problem is that this is not exactly the same as running `http_error` (which will check for `>= 400`).

I guess that even if this is not strictly necessary, I would still rather err on the side of being clearer and more verbose on what the behavior is for the benefit of package users that won't read the source code.